cpu/stm32f1/lpm_arch.c

```diff
@@ -1,6 +1,5 @@
 /*
- * Copyright (C) 2013 INRIA
- * Copyright (C) 2014 Freie Universität Berlin
```

Comment:
@lebrush in any case it's 2016 now and your name is still missing from the list of copyright holders.